qds_sdk/pipelines.py

```diff
@@ -339,23 +339,70 @@ def create(cls, pipeline_name, create_type, **kwargs):
 
         Args:
             pipeline_name: Name to be given.
-            create_type: 1->Assisted, 2->Code, 3->Jar
+            create_type: 1->Assisted, 2->Jar, 3->Code
             **kwargs: keyword arguments specific to create type
 
         Returns:
             response
         """
         conn = Qubole.agent()
-        data = {"data": {
-            "attributes":
-                {"name": pipeline_name, "status": "DRAFT",
-                 "create_type": create_type},
-            "type": "pipelines"}
-        }
-        url = Pipelines.rest_entity_path + "?mode=wizard"
+        url = Pipelines.rest_entity_path
+        if create_type is None:
+            raise ParseError("Provide create_type for Pipeline.", None)
+        if not kwargs or create_type == 1:
+            data = {
+                "data": {
+                        "attributes": {
+                            "name": pipeline_name,
+                            "status": "DRAFT",
+                            "create_type": create_type
+                        },
+                        "type": "pipeline"
+                    }
+            }
+            url = url + "?mode=wizard"
+        else:
+            data = {
+                "data": {
+                    "type": "pipeline",
+                    "attributes": {
+                        "name": pipeline_name,
+                        "create_type": create_type,
+                        "properties": {
+                            "cluster_label": kwargs['cluster_label'],
+                            "can_retry": kwargs['can_retry'],
+                            "command_line_options": kwargs['command_line_options'],
+                            "user_arguments": kwargs['user_arguments']
+                        }
+                    },
+                    "relationships": {
+                        "alerts": {
+                            "data": {
+                                "type": "pipeline/alerts",
+                                "attributes": {
+                                    "can_notify": kwargs['can_notify'],
+                                    "notification_channels": kwargs['channel_ids']
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            if create_type == 2:
+                data['data']['attributes']['properties']['jar_path'] = \
+                    kwargs['jar_path']
```

Comment:
try `kwargs.get` and handle with proper error in case the keys are not present, otherwise, we might. see key errors that the user might not understand

Reply:
We can use `kwargs.get` but I don't think we need to handle the case if keys are present or not over here. That should be done at the QDS API level.

Reply:
yeah error from api also works